### ISSUE TITLE
Windows 11 22H2 (Moment 2): redesigned system tray overflow window cannot be interacted with using mouse and/or touch

### ISSUE THREAD

**Issue by josephsl** [nvaccess/nvda#14539]
Hi,

Follow-up to #14538 but won't show up for a while (Insider Preview feature at the moment):

### Background:
As part of Windows 11 22H2 Moment 2 (under testing), redesigned taskbar and system tray will be made available to users, more so if using tablets such as two-in-one devices. At least taskbar can be navigated using mouse and/or touch, but the redesigned system tray overflow window cannot (at the moment).

### Steps to reproduce:
Prerequisites: Windows 11 22H2 beta builds (one must be a Windows Insider to test this):

1. Install Windows 11 22H2 build 22623 or later (must be a Windows Insider subscribed to beta and/or dev channel).
2. Enable mouse tracking (NVDA+M) and follow mouse from review cursor settings panel.
3. Press Windows+B to move focus to system tray/show hidden icons button.
4. Press Space to activate it.
5. Use the mouse to navigate the overflow window.

### Actual behavior:
No announcement from NVDA regarding overflow buttons.

### Expected behavior:
NVDA announces overflow buttons when moving the mouse.

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:
alpha-27514

#### Windows version:
Windows 11 22H2 beta (10.0.22623)

#### Name and version of other software in use when reproducing the issue:
None

#### Other information about your system:
This is an Insider Preview machine

#### Does the issue still occur after restarting your computer?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Yes - reproducible in 2022.4

#### If NVDA add-ons are disabled, is your problem still occurring?
Yes

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
N/A

### Solution:
Treat the top-level window of the redesigned system tray overflow window as good UIA window (window class name: TopLevelWindowForOverflowXamlIsland). Windows App Essentials add-on includes this solution.

**Comment by josephsl** [comment]
Hi,

Two pull request options:

1. A combined pull request that also resolves #14538
2. A separate pull request just for resolving this issue once Moment 2 comes to release preview channel

The combined pull request can simplify the resulting changelog and get the Moment 2 change ready for 2023.1, while the separate pull request approach can anticipate possible late changes to redesigned system tray overflow menu by the time it comes to release preview. I'm leaning toward the former at the moment as the system tray redesign is here to stay unless Microsoft changes things in January and February.

Thanks.
